ffig/templates/ffig.macros

```diff
@@ -13,10 +16,14 @@
  #       {%- endcall -%}
  #   )
  #}
-{%- macro comma_separated_list(items, leading_comma=False, trailing_comma=False) -%}
+{%- macro comma_separated_list(items, leading_comma=False, trailing_comma=False, extraParam=None) -%}
```

Comment:
This is a non-breaking addition to the shared macro, to support passing an additional parameter back to the callback. In the Go-specific macros, it's used to pass the `impl` class through to the filters. Nothing else makes use of it, yet.

Reply:
I'm not sure I understand the need for this. One can add extra parameters to the end by just adding them to the end (we do so in a few places already) and this extension, while neater, will only add support for a single extra parameter.

I suspect I'm missing something.

Reply:
Ah. I didn't have the jinja2 documentation to hand when I wrote this, so I assumed I needed to name a parameter so I could refer to it later. Turns out I can do this with `varargs`. I'll make the change.